coding.go

```diff
@@ -1,15 +1,18 @@
 package merkledag
 
 import (
+	"bytes"
 	"fmt"
 	"sort"
 	"strings"
 
 	blocks "github.com/ipfs/go-block-format"
-	pb "github.com/ipfs/go-merkledag/pb"
-
 	cid "github.com/ipfs/go-cid"
-	ipld "github.com/ipfs/go-ipld-format"
+	format "github.com/ipfs/go-ipld-format"
+	pb "github.com/ipfs/go-merkledag/pb"
+	dagpb "github.com/ipld/go-ipld-prime-proto"
+	"github.com/ipld/go-ipld-prime/fluent"
```

Comment:
@warpfork - is fluent the right dialect of prime to be using here?

Reply:
I suspect it is -- fluent is basically the way to do a bunch of stuff without an incredibly verbose syntax.

Reply:
You can also try either [`quip`](https://pkg.go.dev/github.com/ipld/go-ipld-prime/fluent/quip) or [`qp`](https://pkg.go.dev/github.com/ipld/go-ipld-prime/fluent/qp) dialects, if you like -- they're questing for good syntax but both should have a lower performance overhead than the `fluent` package does.

`quip` and `qp` are more recently introduced than `fluent`, and someday we'll probably try to narrow down how many of these there are, but for now all three are considered valid (and battle-testing so we can get hands-on feedback about the ergonomics is highly welcome :)).

Reply:
I think that was my question: should we prefer quip / qp vs fluent for this dev?

Reply:
AND I was unaware of quip/qp so now I need ot check those out :)